.github/workflows/label-issue-open.yml

```
@@ -30,3 +41,27 @@ jobs:
         if: ${{ steps.latest-issue.outputs.hasLabels == 'false'}}
         with:
           labels: 'status: awaiting triage'
+
+  community:
+    name: Add community contribution label
+    runs-on: ubuntu-20.04
+    permissions:
+      contents: read
+      pull-requests: write
+      issues: write
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/github-script@v6
+
+      - name: Check if user is a community contributor
+        id: check
+        run: node .github/workflows/scripts/is-community-contributor.js
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: 'If community PR, assign a reviewer'
+        if: github.event.pull_request && steps.check.outputs.is-community == 'yes'
```

```suggestion
        if: steps.check.outputs.is-community == 'yes'
```

Let's move the pull request check to the job itself so that we don't run any of the code here.

---

`is-community-contributor.js` in the previous step adds the community contributor label to both issues & PRs. We want to keep that part of the process.

---

Oh! I didn't realize we assigned community labels to issues too, I thought that was only for pull requests. Carry on :smile:

---

> Do you have a fork with this merged and a link to example issues where it's working?

No, but the labelling missed while working on this PR can be done relatively easily manually (which I have been doing).